content/en/contribute/medic/onboarding/all-the-things.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "All The Things"
+linkTitle: "All The Things"
+weight: 2
+description: >
+  Wide range of topics and resources to support the onboarding
+---
+
+This page is meant to serve as a point of conversation, with a wide range of topics to be discussed when joining Medic or starting as a contributor. Many things are not in any particular order. The goal is to convey a general “lay of the land” so someone starting can see a lot of what’s out there without having to be surprised each day as new things pop up.
+
+## First, some general context…
+
+### What is the CHT?
+* Why the [CHT (Community Health Toolkit)]({{% ref "/why-the-cht" %}})?
+* The [CHT Core Framework]({{% ref "/core" %}}) - Web application that runs inside an Android app (as a [WebView](https://developer.android.com/reference/android/webkit/WebView))
+* [Offline-First]({{% ref "/core/overview/offline-first" %}}) - for real
+* [Community](https://communityhealthtoolkit.org/) of people and organizations
+
+### Teams at Medic
+Meet the [Medic team](https://medic.org/team/)!
+* Product
+* Programs
+* Research
+* Internal Operations
+* External Affairs
+
+### Lifecycle of a [CHT Application]({{% ref "/apps" %}}) being built
+1. Programs team starts relationship with an organization
+1. Service designers and app developers figure out how they want their system to work
+1. App developers take latest version of the CHT and build the app for the organization
+1. Android flavor deployed to get branded app onto CHW devices / deployment
+1. Dashboards are set up in Klipfolio, Superset or Grafana
+1. Go!
+1. App developers make ongoing enhancements
+1. App upgraded as new versions of CHT are available
+
+## Now, all the things…
+### People/Team
+
+#### Distributed things 
+* Without some effort, it’s easy for things to feel lonely or isolated
+* Default to asynchronous things
+* Respect teammate timezones (including your own!)
+
+#### Expensify
+* Follow up to make sure things get through 
+
+#### Funding
+* Restricted - Clicktime
+* Unrestricted
```

Comment:
Some thing like: "Getting our clicktime submitted on time is vital to get source the projects adequately, if necessary take extra steps to remind yourself of submitting these. If you have any question about to fill the reports properly, reach out to the finance team"